fediseer/observer.py

```diff
@@ -64,10 +64,17 @@ def retrieve_suspicious_instances(activity_suspicion = 20, active_suspicious = 5
                 continue
             if local_activity == 0:
                 local_activity= 1
+
+            # posts+comments could be much lower than total users
             if node["total_users"] / local_activity > activity_suspicion:
                 is_bad = True
                 # print(node)
 
+            # posts+comments could be much higher than total users
+            if local_activity / node["total_users"] < activity_suspicion_low:
```

Comment:
needs to be greater than

Reply:
It was a late night! 😴 I hope that's better now :D